Assets/_Project/Scripts/UserInterface/Views/StationView.cs

```diff
@@ -0,0 +1,155 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using PolSl.UrbanHealthPath.UserInterface.Components;
+using PolSl.UrbanHealthPath.UserInterface.Initializers;
+using PolSl.UrbanHealthPath.UserInterface.Interfaces;
+using PolSl.UrbanHealthPath.UserInterface.Popups;
+using TMPro;
+using UnityEngine;
+using UnityEngine.Events;
+using UnityEngine.UI;
+
+namespace PolSl.UrbanHealthPath.UserInterface.Views
+{
+    [RequireComponent(typeof(RectTransform))]
+    public class StationView : MonoBehaviour, IInitializable, IPopupable, IDisplayable
+    {
+        public RectTransform PopupArea => popupArea;
+
+        [SerializeField] private Button rightButton;
```

Comment:
This class is definitely too big, extract smaller components.

Reply:
Too time costly to split it into smaller classes. Changed the fields into arrays of their object types.